saos-restful-web-api/src/test/java/pl/edu/icm/saos/api/dump/enrichmenttag/DumpEnrichmentTagControllerTest.java

```diff
@@ -83,6 +83,7 @@ public void setUp(){
         testObjectContext = testPersistenceObjectFactory.createTestObjectContext();
         enrichmentTags = testPersistenceObjectFactory.createEnrichmentTagsForJudgment(testObjectContext.getCcJudgmentId());
         
+        parametersExtractor.setMinPageSize(1);
```

Comment:
You are changing the bean value here - this can influence other tests that use it.